pallets/pooled-staking/src/pools.rs

```diff
@@ -361,6 +366,34 @@ impl<T: Config> ManualRewards<T> {
         Ok(Stake(diff.err_mul(&shares.0)?))
     }
 
+    /// Increase the rewards of the ManualRewards pool with best effort.
+    /// Returns the actual amount distributed (after rounding).
+    pub fn increase_rewards(
+        candidate: &Candidate<T>,
+        rewards: Stake<T::Balance>,
+    ) -> Result<Stake<T::Balance>, Error<T>> {
+        let Shares(supply) = Self::shares_supply(&candidate);
+        if supply.is_zero() {
+            return Ok(Stake(Zero::zero()));
+        }
+
+        let rewards_per_share = rewards
+            .0
+            .checked_div(&supply)
```

Comment:
It is the opposite, it should be at least equal to the supply of shares to distribute 1 unit per share.

That's why `T::InitialManualClaimShareValue` should be big enough so that the supply is somewhat low and allow rewards to be properly distributed.